app/forms.py

```diff
@@ -0,0 +1,29 @@
+from flask_wtf import FlaskForm
+from wtforms import StringField, FloatField, FileField, TextField, MultipleFileField, TextAreaField
```

Comment:
I think flake8 prevents multi imports, can you install the pre-commit hooks using 'pre-commit install',

Reply:
I'll do it, thanks